go/vt/sqlparser/ast.go

```diff
@@ -1045,6 +1048,22 @@ func (ct *ColumnType) SQLType() querypb.Type {
 		return sqltypes.Set
 	case keywordStrings[JSON]:
 		return sqltypes.TypeJSON
+	case keywordStrings[GEOMETRY]:
+		return sqltypes.Geometry
+	case keywordStrings[POINT]:
+		return sqltypes.Geometry
+	case keywordStrings[LINESTRING]:
+		return sqltypes.Geometry
+	case keywordStrings[POLYGON]:
+		return sqltypes.Geometry
+	case keywordStrings[GEOMETRYCOLLECTION]:
+		return sqltypes.Geometry
+	case keywordStrings[MULTIPOINT]:
+		return sqltypes.Geometry
+	case keywordStrings[MULTILINESTRING]:
+		return sqltypes.Geometry
+	case keywordStrings[MULTIPOLYGON]:
+		return sqltypes.Geometry
```

Comment:
@demmer: I think this big switch can be avoided by:
1. `switch keywords[ct.Type]`
2. Building a `map[int]querypb.Type`

Not a blocker for this.